integration-tests/e2e_tests_ci_tool/cmd/test_config_cmd_test.go

```diff
@@ -0,0 +1,74 @@
+package cmd
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/pelletier/go-toml/v2"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+
+	ctf_config "github.com/smartcontractkit/chainlink-testing-framework/config"
+)
+
+func TestCreateTestConfigCmd(t *testing.T) {
+	tests := []struct {
+		name    string
+		args    []string
+		want    interface{}
+		check   func(t *testing.T, tc *ctf_config.TestConfig)
+		wantErr bool
+	}{
+		{
+			name: "LoggingLogTargets",
+			args: []string{"create", "--logging-log-targets=target1", "--logging-log-targets=target2"},
+			check: func(t *testing.T, tc *ctf_config.TestConfig) {
+				assert.NotNil(t, tc.Logging)
+				assert.NotNil(t, tc.Logging.LogStream)
+				assert.Equal(t, []string{"target1", "target2"}, tc.Logging.LogStream.LogTargets)
+			},
+		},
+		{
+			name: "PrivateEthereumNetworkExecutionLayerFlag",
+			args: []string{"create", "--private-ethereum-network-execution-layer=geth", "--private-ethereum-network-ethereum-version=1.10.0"},
```

Comment:
this doesn't look okay. there are 2 ethereum version supported: `eth1` and `eth2`, roughy `pow` and `pos`. It has nothing to do with image version :-)